app/services/usps_in_person_proofing/transliterable_validator.rb

```diff
@@ -42,12 +42,19 @@ def validate(record)
         invalid_chars = get_invalid_chars(value)
         next unless invalid_chars.present?
 
+        nontransliterable_chars += invalid_chars
+
         record.errors.add(
           field,
           :nontransliterable_field,
           message: get_error_message(invalid_chars),
         )
       end
+
+      if nontransliterable_chars.present?
+        nontransliterable_chars = nontransliterable_chars.sort.uniq
+        log_nontransliterable_characters(nontransliterable_chars)
+      end
```

Comment:
could simplify if we moved to be a set
```suggestion
      if nontransliterable_chars.present?
        log_nontransliterable_characters(nontransliterable_chars.sort)
      end
```